language-service/language_service/service/definition/__init__.py

```diff
@@ -2,63 +2,57 @@
 Where you put definition business logic.
 We combine different data sources as available to deliver the best definitions.
 """
-from language_service.dto import ChineseDefinition
-from language_service.service.definition.cedict import CEDICT
-from language_service.service.definition.wiktionary import Wiktionary
+from multiprocessing.context import TimeoutError
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'TimeoutError'](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5138642)